app/src/main/java/fr/gaulupeau/apps/Poche/events/SyncQueueFinishedEvent.java

```diff
@@ -6,6 +6,7 @@
 public class SyncQueueFinishedEvent extends BackgroundOperationEvent {
 
     protected ActionResult result;
+    protected Long queueLength;
```

Comment:
I use object types instead of primitives types when I need `null` to be a possible value (like "the value is not set"). Otherwise instead of `null` I would probably need to use some "special values" like `-1` which may not be always obvious to use in conditions. `value != null` is pretty much obvious no matter what.
When I'm sure the value is always set, I stick with primitive types.